README.md

```diff
@@ -1 +1,5 @@
-# best-repo-ever
\ No newline at end of file
+# best-repo-ever
+Does it matter how I insert text in the .md readme?
+Don't know yet?
+Is this part of the Pull?
+So now, how far does this get without the commit on GitHub Desktop?
```

Comment:
So each commit is recorded, and all file changes are rolled up into one at this time, let me try and change something further back and see what it does?

Reply:
This is a review!